### PR TITLE
Added Polish translations

### PR DESCRIPTION
**Pull request by strebski**

### CODE REVIEW

**Comment by mpimenov** [comment]
Hello. Please sign the CLA: https://maps.me/cla/

File: data/strings/partners_strings.txt [hunk 1]
```diff
@@ -2,14 +2,17 @@
 
   [sponsored_partner1_action]
     en = Outlets
+    pl = Wyprzedaże
```
Comment (mpimenov) [inline data/strings/partners_strings.txt]: The order in our strings files is: en, then ru, then others.

**Comment by strebski** [comment]
CLA signed.
Thanks for clarify the rule :-)

**Comment by mpimenov** [comment]
We are experiencing problems with our spam filters. Could you please sign it once more?

**Comment by strebski** [comment]
Done.

File: data/strings/partners_strings.txt (hunk 1 above)
Comment (mpimenov) [reply data/strings/partners_strings.txt]: ping on this comment

Comment (mpimenov) [reply data/strings/partners_strings.txt]: ping

**Pull request merged**